src/cli/parseArgs.js

```diff
@@ -29,18 +28,20 @@ function parseArgs() {
     return true;
   });
 
-  program.version('1.0.1')
+  program.version(packageInfo.version)
     .option(
       '-c, --huron-config [huronConfig]',
-      '[huronConfig] for all huron options',
-      path.resolve(__dirname, '../defaultConfig/huron.config.js')
+      '[huronConfig] for all huron options'
     )
     .option(
       '-w, --webpack-config [webpackConfig]',
-      '[webpackConfig] for all webpack options',
-      path.resolve(__dirname, '../defaultConfig/webpack.config.js')
+      '[webpackConfig] for all webpack options'
     )
-    .option('-p, --production', 'compile assets once for production');
+    .option('-p, --production', 'compile assets once for production')
+    .option(
+      '--use-prototype [usePrototype]',
```

Comment:
Would adding `-u` conflict with anything?

Reply:
nope, I can add that as the shortcut